teacher.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- Great Job for extends parent's constructor by adding @specialization however you forget to add the parameters for it so please add it to fit the requirment👍
```
def initialize(age, specialization, name = "Unknown", parent_permission = true)
```